.flowconfig

```diff
@@ -5,6 +5,7 @@ include_warnings=true
 module.name_mapper='^firefox-profiler\/\(.*\)$' -> '<PROJECT_ROOT>/src/\1'
 module.name_mapper='^firefox-profiler-res\/\(.*\)$' -> '<PROJECT_ROOT>/res/\1'
 module.name_mapper.extension='ftl' -> '<PROJECT_ROOT>/src/types/mocks/ftl.js'
+esproposal.optional_chaining=enable
```

Comment:
Note that we still can't use it in the app code because of our using of webpack v4. But at least this works for our tests.

Reply:
I really want to use it in the app source code too 😄 I hope we can upgrade to v5 soon.